robot/rover/MobilePlatform/includes/commands/WheelsCommandCenter.h

```diff
@@ -1,23 +1,16 @@
-//
-// Created by Michael on 2021-01-23.
-//
-
 #ifndef WHEELS_WHEELSCOMMANDCENTER_H
 #define WHEELS_WHEELSCOMMANDCENTER_H
 
-#include "../../../internal_comms/include/CommandCenter.h"
-
-#define COMMAND_SET_MOTORS 0
-#define COMMAND_STOP_MOTORS_EMERGENCY 1
-#define COMMAND_CLOSE_MOTORS_LOOP 2
-#define COMMAND_OPEN_MOTORS_LOOP 3
-#define COMMAND_SET_JOYSTICK 4
-#define COMMAND_SET_GPS 5
-#define COMMAND_SET_ENCODER 6
-#define COMMAND_SET_ACCELERATION 7
-#define COMMAND_GET_ROVER_STATUS 8
-#define COMMAND_MOVE_ROVER 9
-#define COMMAND_MOVE_WHEEL 10
+#include "CommandCenter.h"
+
+#define COMMAND_STOP_MOTORS_EMERGENCY 0
+#define COMMAND_GET_ROVER_STATUS 2
+#define COMMAND_SEND_GPS 3
+#define COMMAND_MOVE_SERVO 7
```

Comment:
Can you please change all the command IDs so that they are sequential? And reflect the changes in WheelCommands.py

Reply:
I will remove all the un-used commands until they actually work (like the velocity stuff until encoders work again)